upstream-tools/replacements.json

```diff
@@ -1849,11 +1881,29 @@
       "new": "Resource for managing an AWS NetworkManager ConnectAttachment."
     }
   ],
+  "website/docs/r/networkmanager_connect_peer.html.markdown": [
+    {
+      "old": "  Terraform resource for managing an AWS NetworkManager Connect Peer.",
```

Comment:
Another (near-)duplicate entry. Are both of these needed?

Reply:
Yeah, only the second should be required here, but won't cause any issue having both right now.